spec/features/system/admin/user_management_spec.rb

```diff
@@ -10,7 +10,7 @@
 
       let!(:users) do
         create_list(:user, 2)
-        User.ordered_by_name.page(1)
+        User.human_users.ordered_by_name.page(1)
```

Comment:
@lowjoel How about just name this scope `human`, I think that's more conventional ?